### ISSUE TITLE
Issue with Wacom tablet

### ISSUE THREAD

**Issue by naatje80** [TurboVNC/turbovnc#220]
We are currently encountering issues with various Wacom tablets. I already read this is a work in progress. But in fact, at the moment we are not interested in Wacom support itself. Is there a possibility to disable wintab library support either on run or compile time?  We noticed that if we remove the driver, the tablet just behaves like a normal mouse. But this also disables some features they could be useful in Windows itself. I already had a quick look in the code myself, but it currently seems to take a lot of work to remove this specific code. Did we overlook a certain option?

**Comment by dcommander** [comment]
Yeah, WinTab unfortunately does not support the most recent Wacom tablets.  Supporting those tablets would require using the Windows Ink API (see https://github.com/TurboVNC/turbovnc/issues/146), and since Wacom support was a funded feature, I'm not interested in extending it unless someone pays for my labor.  Windows Ink support is also a feature that would need to land in the new unified TurboVNC Viewer, which will be introduced in TurboVNC 3.0, rather than in the deprecated Windows TurboVNC Viewer.  

For the moment, I've modified the Windows TurboVNC Viewer such that it disables Wacom/WinTab support if the environment variable `TVNC_WINTAB` is set to `0`.  Unfortunately, you will have to use the 2.2.x pre-release build to get this functionality, since 2.2.5 has already been released.
